keyboards/rookiebwoy/late9/rev1/info.json

```diff
@@ -19,6 +19,10 @@
         "rows": ["B4", "E6", "D7", "B3", "B2", "B6"]
     },
     "diode_direction": "ROW2COL",
+    "features": {
+      "bootmagic": true,
+      "nkro": true
```

Comment:
```suggestion
      "nkro": true,
      "oled": true
```

Reply:
Done. I think that a list of possible features in the documentation could be useful.